framework/lib/components/flip-button/flip-button-group.tsx

```diff
@@ -17,6 +17,7 @@ export const FlipButtonGroup = (props: FlipButtonGroupProps) => {
     children,
     direction = 'row',
     isDisabled = false,
+    hasCheckmark = true,
```

Comment:
Also need it on FlipBUttonGroupField.

Reply:
This is actually not a bad idea

Reply:
Sorry i missed that one. 

`showIcon: 'left' | 'rigth' | 'none' ` seems pretty understandable to me.

Reply:
@ilija- check latest
`iconPos: 'left' | 'right' | 'none'`

Reply:
Position none ?

Reply:
showIcon implies action so it should be a function. @ilija-

Reply:
hmm .. then I'm okay with that. `iconPos` .. seems plausible.